src/components/pill/pill.scss

```diff
@@ -3,14 +3,46 @@
 .carbon-pill {
   border-radius: 10px;
   display: inline-block;
-  font-size: 11px;
+  font-size: 12px;
   font-weight: 700;
-  line-height: 12px;
   letter-spacing: 1px;
   padding: 2px 7px;
   text-align: center;
 }
 
+.carbon-pill_with-icon {
+  font-size: 14px;
+  line-height: 14px;
+}
+
+.carbon-pill__delete-icon {
+  margin-left: 5px;
+  margin-right: -5px;
+}
+.carbon-pill__delete-icon:before {
+  font: {
```

Comment:
can this just be font-size?

Reply:
Yeah I realised that after I pushed it, have changed it